src/transactionsLoopV2/serverless.yaml

```diff
@@ -0,0 +1,45 @@
+service: transactionsLoopV2
+
+package:
+  artifact: dist/index.zip
+
+plugins:
+  - serverless-offline
+
+provider:
+  name: aws
+  runtime: nodejs18.x
+  memorySize: 1028
+  timeout: 900
+
+functions:
+  memesTransactionsLoop:
+    handler: index.handler
+    name: memesTransactionsLoop
+    description: ${env:VERSION_DESCRIPTION}
+    role: arn:aws:iam::987989283142:role/lambda-vpc-role
+    reservedConcurrency: 1
+    events:
+      - schedule: cron(1 0 * * ? *)
```

Comment:
is this schedule right? is it every minute?
why not use the more readable `rate(x minutes)`

Reply:
You are right. It is not correct. I thought I copied this from transactionsLoop, but it was from tdhLoop. Fixed it.